.github/workflows/delete_review_app.yml

```diff
@@ -9,44 +9,45 @@ on:
       - 'terraform/common/**'
       - '**.md'
   workflow_dispatch:
+  # Wait until the Build And Deploy for the same PR is finished before starting this workflow.
+  # If triggered by a manual workflow dispatch with a PR number, builds the corresponding 'ref' to match the Build and Deploy concurrency group for that PR.
+  # For a closed or unlabeled PR, the 'ref' is available and matches the Build and deploy one.
```

Comment:
This comment was referencing the following concurrency definition that is being removed:
`concurrency: workflow-Build-and-deploy-${{ (github.event_name == 'workflow_dispatch' && format('refs/pull/{0}/merge', github.event.inputs.pr_number)) || github.ref }}`

Since this has been replaced (we do not hold for deploys anymore?), the above comment became irrelevant now :axe: